README.md

```diff
@@ -51,6 +51,18 @@ Community (college) maintained list of Algorithms and Data Structures implementa
 | [Stack](https://en.wikipedia.org/wiki/Stack_(abstract_data_type)) | | | [\[X\]](stack/Stack.java) | [\[X\]](stack/stack.py) | [\[X\]](stack/stack.go) | [\[X\]](stack/stack.js) |
 
 
+## How to run them
+
+| Language | Steps |
+|:--------------|:----------------|
+| C |<pre>gcc [filename.c]<br>./a.out  # unix<br>a.exe  # windows</pre>|
+| CPP |<pre>g++ [filename.cpp]<br>./a.out # unix<br>a.exe # windows</pre>|
```

Comment:
Please Change `a.exe`  to `./a.exe` as they are different and one isn't working
![screenshot 21](https://user-images.githubusercontent.com/21199234/27039301-3b08bb86-4fab-11e7-9091-ba8984b2ae5a.png)